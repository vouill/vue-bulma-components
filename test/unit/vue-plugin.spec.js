import vbc, { bulmaComponentGenerator } from 'src/plugin/plugin'

describe('Bulma component generator ', () => {
  it('should access bulma componentized plugin', () => {
    expect(vbc).toMatchSnapshot()
  })
  it('should access the bulma component generator func ', () => {
    expect(bulmaComponentGenerator).toMatchSnapshot()
  })
})
