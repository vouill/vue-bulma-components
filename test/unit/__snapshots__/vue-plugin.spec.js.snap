// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bulma component generator  should access bulma componentized plugin 1`] = `
Object {
  "install": [Function],
}
`;

exports[`Bulma component generator  should access the bulma component generator func  1`] = `[Function]`;
